src/model/ControllerMechanicalVentilation.cpp

```diff
@@ -109,8 +109,13 @@ namespace model {
           if (oaSystem) {
             const auto airLoop = oaSystem->airLoopHVAC();
             if (airLoop) {
-              const auto sizingSystem = airLoop->sizingSystem();
-              result = sizingSystem.systemOutdoorAirMethod();
+              const auto sizingSystemOAMethod = airLoop->sizingSystem().systemOutdoorAirMethod();
+              const auto possible_vals = systemOutdoorAirMethodValues();
+              if (std::find_if(possible_vals.cbegin(), possible_vals.cend(),
+                               [&sizingSystemOAMethod](const std::string& val) { return openstudio::istringEqual(val, sizingSystemOAMethod); })
+                  != possible_vals.cend()) {
+                result = sizingSystemOAMethod;
+              }
```

Comment:
Simple fix: do not inherit the SizingSystem's System Outdoor Air Method value if this option isn't one available in Controller:MV.

![image](https://github.com/user-attachments/assets/33d9451e-b37a-4837-a979-826f6df7ac8a)